### PR TITLE
Add konflux configuration for next version

### PR DESCRIPTION
**Pull request by pramodbindal**

### CODE REVIEW

File: config/konflux/tektoncd-cli.yaml [hunk 1]
```diff
@@ -1,4 +1,7 @@
 repository: tektoncd-cli
 upstream: tektoncd/cli
 components:
-- tkn
\ No newline at end of file
+  - tkn
+branches:
+  - upstream: release-v0.65.x
```
Comment (savitaashture) [inline config/konflux/tektoncd-cli.yaml:6]: why 65.x for cli
i thinks its different version can you check and change

File: config/konflux/tektoncd-hub.yaml [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
 upstream: tektoncd/hub
 components:
-- db-migration
-- hub-api
-- hub-ui
+  - db-migration
+  - hub-api
+  - hub-ui
+branches:
+  - upstream: release-v0.65.x
```
Comment (savitaashture) [inline config/konflux/tektoncd-hub.yaml:8]: same fro hub check the hub released version

File: config/konflux/tektoncd-triggers.yaml [hunk 3]
```diff
@@ -1,7 +1,11 @@
...
-- controller
-- webhook
-- core-interceptors
-- eventlistenersink
+  - controller
+  - webhook
+  - core-interceptors
+  - eventlistenersink
+branches:
+  - upstream: release-v0.65.x
```
[the remark this answers is not in the document]
Comment (pramodbindal) [reply config/konflux/tektoncd-triggers.yaml]: Done

File: config/konflux/tektoncd-hub.yaml (hunk 2 above)
Comment (pramodbindal) [reply config/konflux/tektoncd-hub.yaml]: Done

File: config/konflux/tektoncd-cli.yaml (hunk 1 above)
Comment (pramodbindal) [reply config/konflux/tektoncd-cli.yaml]: ohh, i thought tekton might be following same release version for all components

Comment (pramodbindal) [reply config/konflux/tektoncd-cli.yaml]: done

**Review by vdemeester — APPROVED** [review]

**Review by savitaashture — COMMENTED** [review]
/lgtm

**Pull request merged**